packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -29,7 +29,7 @@ export type FormFieldConfig<T> = (
           filterField?: { name: string; gqlVarName: string; gqlVarType: "rootProp" | "filter" };
       }
     | { type: "block"; block: ImportReference }
-) & { name: keyof T; label?: string; required?: boolean; validate?: ImportReference; helperText?: string; readOnly?: boolean };
+) & { name: keyof T; label?: string; required?: boolean; virtual?: boolean; validate?: ImportReference; helperText?: string; readOnly?: boolean };
```

Comment:
as an alternative this could be done implicit only for asyncSelects with graphql-alias, for now this is the only use-case I can think of requiring no saving...

is there any use case readonly-fields need to be saved/sent? if not it could also be omitted implicitly.

Reply:
another alternative could be to check if the input type contains the field

Reply:
you're right. so the main question for now is config vs detection. probably still preferre detection because it reduces possible mistakes and auto-correctes admin if api changes.

Reply:
argument for detection: easier to use, simpler (no) Api

argument for config: more explicit, especially for such an feature (form field that does **not** get saved) this might be preferable

Reply:
for now we don't have a requirement to explicit disable save. @johnnyomair should we stick with the setting or do it implicit (if there is no input for field-name)

Reply:
I don't have a too strong opinion but would decide for config

Reply:
Yes, make it explicit